projects/documentation/content/support-and-compatibility.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: guide.njk
+title: 'Support and compatibility: Spectrum Web Components'
+displayName: Support and compatibility
+slug: support-and-compatibility
+---
+
+# Support and compatibility: Spectrum Web Components
+
+This page provides comprehensive information on versioning, public APIs, browser support, and issue severity classification to ensure a seamless integration experience for developers. Below, you'll find detailed guidelines and policies to help you understand our commitment to maintaining the stability and compatibility of Spectrum Web Components.
+
+## Versioning
+
+Starting from version 1.0.0, Spectrum Web Components strictly follows semantic versioning ([semver](https://semver.org/)). We regularly release patch versions, which do not contain breaking changes. When a breaking change occurs, it will be done in a major version release to avoid breaking existing applications depending on the old version. Major version releases will be communicated in advance, and migration guides will be provided.
+
+## Public APIs
+
+Our public API consists of:
+
+-   Component APIs (properties, attributes, slots, events, functions)
+-   TypeScript definitions
+-   File import paths
+
+## Browser Support
+
+We support the latest 2 major versions of these browsers for desktop:
+
+-   Google Chrome
+-   Mozilla Firefox
+-   Microsoft Edge
+-   Apple Safari for macOS
+
+We do not support Microsoft Internet Explorer.
+
+We support all viewport sizes across supported desktop browsers. While our components are designed to be responsive and mobile-friendly, we do not yet fully support mobile browsers due to limited testing on mobile hardware. We advise testing updates on mobile devices before updating and are happy to address any reported issues.
+
+## Reporting bugs
```

Comment:
Not sure how i feel about including the bug report information in this file... it seems like it's not relevant to "Support and compatibility", and might go unseen because the title of the document doesn't indicate that it would include bug reporting. 

I would recommend either removing it and putting it in its own file under the "Contributing" guides section, or just not including it at all on our docs site.

Reply:
I was also on the fence about it, and I do not have a strong preference for keeping it. I'll remove!

Reply:
We can revisit this once we've agreed on the path forward (GitHub vs Jira). Depending on the decision, having more information for customers will be probably be helpful in ensuring consistency for our team.